apps/web/src/components/editor/index.tsx

```diff
@@ -59,7 +59,7 @@ export const Editor = memo(
       <div className="overflow-y-auto">
         <EditorRoot>
           <EditorContent
-            className="h-full w-full p-1"
+            className="editor-content h-full w-full p-1"
```

Comment:
if you want underline in links edit the editrProps classlist and add `prose-a:underline` there

Reply:
works. Will do that asap

Reply:
However I wanted cursor style change on hovering over links.

Reply:
![Capture](https://github.com/user-attachments/assets/4f2c5e10-fb8d-4381-8567-14730ef529db)

You mean here right?